lib/dor/text_extraction/ocr.rb

```diff
@@ -8,7 +8,7 @@ class Ocr
 
       def initialize(params = {})
```

Comment:
not necessarily in scope for this PR since it's this way already, but i wonder if there's any particular reason we're using a params hash here instead of named params?

Reply:
1. Good idea on tmp paths in settings - I'll put a small PR to differentiate the input/output/results folders in tmp.
2. The XML ticket path is configured to be at the root of the INPUT path named by druid, which is why it it's own delete step (else it wouldn't get cleaned up by the other actions), see https://github.com/sul-dlss/common-accessioning/blob/main/lib/dor/text_extraction/abbyy/ticket.rb#L16-L22 and https://github.com/sul-dlss/shared_configs/blob/common-accessioning-qa/config/settings/production.yml#L27   Deleting the XML ticket is the last thing that happens, but the order of output folder vs ticket deletion is probably not super important.
3. If the input dir exists but is not empty, it will raise an error in this implementation, blocking everything else from happening by putting the step in error.  I guess once we see how this goes, if it turns out the input folder is not always empty and this is totally fine and acceptable, we could always remove the raise and just let the cleanup continue.
4. Not sure what the deal is with the params hash in the method signature, I'll fix in the same PR to use named params.

Reply:
https://github.com/sul-dlss/common-accessioning/pull/1277

Reply:
> If the input dir exists but is not empty, it will raise an error in this implementation, blocking everything else from happening by putting the step in error. I guess once we see how this goes, if it turns out the input folder is not always empty and this is totally fine and acceptable, we could always remove the raise and just let the cleanup continue.

sounds like a reasonable plan